msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java

```diff
@@ -406,9 +406,15 @@ public void run() {
      * @param homeAccountIdentifier The home_account_id of the sought IAccount.
      * @return The IAccount stored in the cache or null, if no such matching entry exists.
      */
-    public IAccount getAccount(final String homeAccountIdentifier) {
+    public IAccount getAccount(@NonNull final String homeAccountIdentifier,
```

Comment:
If we were >v1.0.0, this would be a API major breaking change. Since we still <1.0.0, I'm feeling empowered to make this change.

Reply:
I've left the `realm` as `@Nullable` since this is IdP specific